docs/UserGuide.md

```diff
@@ -131,19 +131,21 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
```

Comment:
I think we could remove this and replace it with our own example.

Reply:
Done. Thanks Leslie!